plugins/header_rewrite/operators.cc

```diff
@@ -30,6 +30,91 @@
 #include "operators.h"
 #include "ts/apidefs.h"
 
+namespace
+{
+const unsigned int LOCAL_IP_ADDRESS = 0x0100007f;
+const unsigned int MAX_SIZE         = 256;
+const int          LOCAL_PORT       = 8080;
+
+static int
```

Comment:
It doesn't need to be static.  Putting it in an anonymous namespace makes it file scope.